lib/mappers/michigan_mapper.py

```diff
@@ -307,4 +318,17 @@ def map_multiple_fields(self):
         self.map_collection()
         self.map_date_and_publisher()
         self.map_data_provider_and_intermediate_provider()
-        self.map_object_and_is_shown_at()
\ No newline at end of file
+        self.map_object_and_is_shown_at()
+
+    # Get values from property
+    # Returns list of values
+    def get_date(self, s):
+        dates = []
+
+        if not isinstance(getprop(s, "dateIssued"), basestring):
+            for d in iterify(getprop(s, "dateIssued")):
+                dates.append(getprop(d, "#text"))
```

Comment:
@markbreedlove Understood. I added that check for `#text` to address that and a more explicit handling of the `list` case but the whole `map_date` method is a little dirty